src/gui/properties/propertieswidget.cpp

```diff
@@ -517,17 +517,18 @@ void PropertiesWidget::loadUrlSeeds()
     }
 }
 
-void PropertiesWidget::openDoubleClickedFile(const QModelIndex &index)
+void PropertiesWidget::openDoubleClickedFile(QModelIndex index) const
```

Comment:
@Chocobo1, you used to insist that it was bad practice to have non-const parameters...

Reply:
Usually I do that when the original parameter would still have some use, however in this case I think it is better to update the parameter to avoid misuse.

Reply:
> I think it is better to update the parameter to avoid misuse.

To really avoid misuse you shouldn't pass QModelIndex in other methods. Otherwise you should adjust it in these methods.

Reply:
>Otherwise you should adjust it in these methods.

OK, updated.